src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
 export interface Figure {
+  color: string;
+  shape: string;
 
+  getArea(): number;
 }
 
-export class Triangle {
+function isTrianglePossible(a: number, b: number, c:number): boolean {
+  const sum = [a, b, c];
```

Comment:
`sum` is bad name because I guess to see sum of numbers instead (a + b + c). Suggest to rename in `sides` for example